video_xblock/video_xblock.py

```diff
@@ -470,20 +480,25 @@ def _make_field_info(self, field_name, field):
         info = super(VideoXBlock, self)._make_field_info(field_name, field)
         if field_name == 'handout':
             info['type'] = 'file_uploader'
-            info['file_name'] = self.get_handout_file_name()
+            info['file_name'] = self.get_file_name_from_path(self.handout)
             info['value'] = self.get_path_for(self.handout)
         if field_name == 'transcripts':
             info['type'] = 'transcript_uploader'
         return info
 
-    def get_handout_file_name(self):
+    def get_file_name_from_path(self, field):
         """
-        Field handout look like this:
-        asset-v1-RaccoonGang+1+2018+type@asset+block@<filename>
+        Helper for getting filename from string with path to mongoDB storage.
```

Comment:
Good job with docstring.  What if format will change? Is there any standard open edx function that does this?